schemas/rules/operators/value/date.json

```diff
@@ -1,16 +1,25 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "https://eq.ons.gov.uk/rules/operators/value/date-offset.json",
-  "description": "Offsets a date, takes two arguments",
+  "$id": "https://eq.ons.gov.uk/rules/operators/value/date.json",
+  "description": "Casts a value to a date, first argument is a date, second is an offset",
```

Comment:
Would this not pass if I provided just an offset? Also do we ever say arguments are optional in the description?

Reply:
As discussed the date is mandatory as it's the first item in the array. I have updated the text to make it explicit that the offset is optional.